js/limits.js

```diff
@@ -1,2 +1,29 @@
 //Ограничения, накладываемые на поля ввода
 
+const typeSelect = document.querySelector('#type');
+const typeOptions = typeSelect.querySelectorAll('option');
+
+const priceInput = document.querySelector('#price');
+
+
+typeSelect.addEventListener('input', () => {
```

Comment:
Давай все обработчики выносить в отдельные функции. По неймингу мы их называем согласно критерию Д4. В конкретно нашем случае так
```suggestion
const onTypeSelectChange = () => {....}

typeSelect.addEventListener('input', onTypeSelectChange)
```